### DISCUSSION TITLE
my g-helper exe disappears/is missing from my pc

### DISCUSSION THREAD

**Discussion by yjhyuks** [seerge/g-helper#3312] [Q&A]
the extracted folder is empty, and it's also not in the recycle bin. when i search for G-Helper and click the .exe shortcut, it says "the item that this refers to has been changed or moved [...]."

this has happened twice now, and only after startup. it's driving me a little crazy thinking that i'll have to keep re-downloading the zip.
any help please? sorry in advance if this is a dumb question and i just missed reading any obvious descriptions

**Comment by seerge** [comment]
@yjhyuks if file is deleted - it simply means that someone or something has deleted app from your computer :) 
Do you have any strange cleanup utilities or antiviruses running that might do that ?
